### ISSUE TITLE
help me

### ISSUE THREAD

**Issue by aflkalkgjhalk** [legend2ks/YoutubeDownloader#17]
hello
What is the problem?
![Screenshot 2024-06-06 170755](https://github.com/legend2ks/YoutubeDownloader/assets/126336595/0fe43d84-bd3a-4e47-8575-4da3cf976c51)

**Comment by aflkalkgjhalk** [comment]
I click on the plus sign above and copy the link of the video I want from YouTube and paste it there.
But nothing happens

**Comment by legend2ks** [comment]
I added the same video without problem. Did you check the log file?

![image_407](https://github.com/legend2ks/YoutubeDownloader/assets/16824470/93826fb8-496c-4d77-8d29-5ad6d00be6a8)

**Comment by aflkalkgjhalk** [comment]
Thanks for your reply, but I'm a mechanical engineering student and I don't know anything about programming or even GitHub. I also don't know what a log file is. All I did was download your software, install the .NET 8.0 Desktop Runtime, copy a link from YouTube, and paste it into your program, but I got an error. Could you please provide clearer instructions based on what I've said

**Comment by aflkalkgjhalk** [comment]
I found a file in the program folder. If that's what you're referring to, I can upload it for you.
[log20240606.txt](https://github.com/user-attachments/files/15672544/log20240606.txt)

**Comment by aflkalkgjhalk** [comment]
I don't know if this is important, but I live in Iran, and YouTube is filtered here. I have to use a VPN to access YouTube. Does this affect the program's functionality?

**Comment by legend2ks** [comment]
> I don't know if this is important, but I live in Iran, and YouTube is filtered here. I have to use a VPN to access YouTube. Does this affect the program's functionality?

Yes, that should be the problem. Your system needs to have access to YouTube.

**Comment by aflkalkgjhalk** [comment]
"I have access to YouTube and can watch videos using a VPN. I can also download videos one by one from YouTube video download sites with a VPN. Additionally, I can experimentally download videos directly from YouTube itself.

**Comment by legend2ks** [comment]
In your case, this app needs to connect to YouTube via VPN. You need a **system-wide** VPN access.

**Comment by bobbobbin** [comment]
Hi, i was just having this trouble with the latest 3.3, until i added the program to my split tunnel vpn feature, and, i added the firefox cookies - now it seems to work fine, cheers
